cmake/haros_catkin-extras.cmake.em

```diff
@@ -16,9 +23,10 @@ endmacro()
 
 function(haros_report)
   set(HAROS_REPORT_LOCATION "${CMAKE_CURRENT_BINARY_DIR}/test_results/haros_report")
+  set(HAROS_COMMAND "${@(PROJECT_NAME)_VENV_BIN_DIR}/python" "${@(PROJECT_NAME)_VENV_BIN_DIR}/haros")
```

Comment:
Doesn't the `haros` entry point script already spec the absolute path to the venv python after it has been installed with `pip`?

Could be that it doesn't, just wondering.

Reply:
Good point. I'll check it soon.

Reply:
@nlimpert: have you had a chance to check this?

Reply:
This is the contents of `bin/haros` inside the `virtualenv` that `catkin_virtualenv` creates:

```python
#!/home/user/catkin_virtualenv_ws/install/share/haros_catkin/venv/bin/python
# -*- coding: utf-8 -*-
import re
import sys

from haros.haros import main

if __name__ == '__main__':
    sys.argv[0] = re.sub(r'(-script\.pyw?|\.exe)?$', '', sys.argv[0])
    sys.exit(main())
```
looks like we can just invoke the script itself directly and it should do-the-right-thing.